Userland/Libraries/LibJS/Runtime/WeakMap.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2021, Idan Horowitz <idan.horowitz@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/HashMap.h>
+#include <LibJS/Runtime/GlobalObject.h>
+#include <LibJS/Runtime/Object.h>
+#include <LibJS/Runtime/WeakContainer.h>
+
+namespace JS {
+
+class WeakMap : public Object
+    , public WeakContainer {
```

Comment:
The preferred formatting is

```suggestion
class WeakMap
    : public Object
    , public WeakContainer {
```

Also, please make this `final`.

Reply:
fixed